msteams-platform/concepts/Cards/cards-reference.md

```diff
@@ -46,6 +46,159 @@ See the topic [Card Formatting](~/concepts/cards/cards-format) for more informat
 
 ## Cards supported in Teams
 
+### Adaptive card
+
+A customizable card that can contain any combination of text, speech, images, buttons, and input fields.
+
+#### Support
+
+| Bots in Teams | Messaging Extensions  | Connectors | Bot Framework |
+| --- | --- | --- | --- |
+| ✔ | ✔ |  ✔* | ✔ |
+|
+
+#### Example
+
+![Example of a adaptive card card](~/assets/images/cards/adaptivecard.png)
+
+#### For more information
+
+[Adaptive Cards Overview](https://docs.microsoft.com/en-us/adaptive-cards/)
+
+*Note: There are some Outlook specific action types like Action.Http which are not supported by Teams. For more information on Outlook specific behavior of adaptive cards please visit: [Designing Outlook Actionable Message cards with the Adaptive Card format](https://docs.microsoft.com/en-us/outlook/actionable-messages/adaptive-card)
```

Comment:
Should this note be right after the table? It looks a bit off with the 2 intervening sub-sub-sections.